viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -682,14 +580,8 @@ void CartoFacade::GetInternalState(viam_carto_get_internal_state_response *r) {
     }
     boost::uuids::uuid uuid = boost::uuids::random_generator()();
 
-    std::string filename;
-    if (config.cloud_story_enabled) {
-        filename = "temp_internal_state_" + boost::uuids::to_string(uuid) +
-                   ".pbstream";
-    } else {
-        filename = path_to_internal_state + "/" + "temp_internal_state_" +
-                   boost::uuids::to_string(uuid) + ".pbstream";
-    }
+    std::string filename =
+        "temp_internal_state_" + boost::uuids::to_string(uuid) + ".pbstream";
```

Comment:
should this be in a tmp dir?

Reply:
Good point - I think so. @JohnN193 can you take a look and confirm?

Reply:
i think having a temp dir just adds more for us to cleanup, but if its better practice to use one then thats fine with me!

Reply:
I think it is better practice - for linux machines anything in that directory will get cleaned up during boot according to https://linuxhandbook.com/tmp-directory/amp/

This would at less protect us on some machines in the case of a crash before cleanup